LEAF_Request_Portal/templates/print_form.tpl

```diff
@@ -1,8 +1,8 @@
 <!--{if $deleted > 0}-->
-    <div style="font-size: 36px"><img src="dynicons/?img=emblem-unreadable.svg&amp;w=96" alt="Unreadable"
-            style="float: left" /> Notice: This request has been marked as deleted.<br />
+    <div style="font-size: 36px"><img src="dynicons/?img=emblem-unreadable.svg&amp;w=96"
```

Comment:
These alt attributes should just be empty strings (rather than absent)

Reply:
Why does it need an alt tag? Screen readers will just ignore the element if there isn't one.

Reply:
Thanks, updated.